### PR TITLE
Release V1.2 - international release

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [x] Your PR is linked to an issue.
- [x] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [x] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [ ] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR covers issue #361 

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

File: Document-es/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## V1.2 7 March 2020 - International Release
```
Comment (cpholguera) [inline Document-es/CHANGELOG.md]:
```suggestion
## V1.2 7 de marzo 2020 - Release Internacional
```

File: tools/RECENT_CHANGES.TXT [hunk 2]
```diff
@@ -1,3 +1,6 @@
-## V1.2-TestRelease
+# V1.2 International Release
 
-This is a testrelease which will be deleted when the release testing is done.
\ No newline at end of file
+With a little bit of delay we are happy to present version 1.2 of the MASVS!
+Changes:
+- Created international version of V1.2: the MASVS is now translated into German, Spanish, French, Japanese, Korean, Russian, Simplified Chinese, and Traditional Chinese.
+- New buildsystem and release-system based on Github Actions and Docker containers resulting in better looking PDFS, Mobi, Epubs and Docx documents.
```
Comment (cpholguera) [inline tools/RECENT_CHANGES.TXT]:
```suggestion
- New build and release systems based on Github Actions and Docker containers resulting in better looking PDF, Mobi, Epub and Docx documents.
```

**Comment by commjoen** [comment]
> Added Spanish translation and suggestions

Fixed & applied

**Comment by sushi2k** [comment]
LGTM! Thanks 🙏

File: Document-de/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # Versionshistorie
 
+## V1.2 7 March 2020 - International Release
```
Comment (sushi2k) [inline Document-de/CHANGELOG.md]:
```suggestion
## V1.2 7 März 2020 - Internationale Veröffentlichung 
```

Comment (sushi2k) [reply Document-de/CHANGELOG.md]: German Transaktion

File: Document-ko/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
 # 변경이력
 
+## 20 V1.2 2020년 3월7 일-국제 릴리스
```
Comment (OWASP-Seoul) [inline Document-ko/CHANGELOG.md]: ## V1.2 2020년 3월 7일 - 국제 릴리스

Comment (commjoen) [reply Document-ko/CHANGELOG.md]: Thanks! Fixed it!

File: Document-ru/CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,7 @@
 # Список изменений
 
+## V1.2 7 марта 2020 г. - международный релиз
```
Comment (emartynov) [inline Document-ru/CHANGELOG.md]:
```suggestion
## V1.2 - 7 марта 2020 г. - Международный релиз
```

Comment (emartynov) [reply Document-ru/CHANGELOG.md]: Maybe even remove `г.`?

File: Document-ru/CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,7 @@
...
+
 ## V1.2-RC - 5 October 2019 - Предварительный релиз
```
Comment (emartynov) [inline Document-ru/CHANGELOG.md]:
```suggestion
## V1.2-RC - 5 октября 2019 - Предварительный релиз
```

File: Document-ru/CHANGELOG.md (hunk 5 above)
Comment (emartynov) [reply Document-ru/CHANGELOG.md]: Corrected

**Comment by sushi2k** [comment]
Got confirmation from Jack (Simplified chinese) that the translation is good to go

**Pull request merged**